tests/devices/i19/test_pin_col.py

```diff
@@ -0,0 +1,161 @@
+from collections.abc import AsyncGenerator
+from unittest.mock import AsyncMock, MagicMock, call
+
+import pytest
+from ophyd_async.core import init_devices
+from ophyd_async.testing import get_mock_put, set_mock_value
+
+from dodal.devices.i19.pin_col_stages import (
+    AperturePosition,
+    PinColConfiguration,
+    PinColPosition,
+    PinColRequest,
+    PinholeCollimatorControl,
+)
+from dodal.devices.motors import XYStage
+from dodal.testing import patch_all_motors
+
+
+@pytest.fixture
+async def pincol() -> AsyncGenerator[PinholeCollimatorControl]:
+    async with init_devices(mock=True):
+        pc = PinholeCollimatorControl("", "test_pincol")
+    with patch_all_motors(pc):
+        yield pc
+
+
+def test_pincol_created_without_errors():
+    pincol = PinholeCollimatorControl("", "test_pincol")
```

Comment:
Should: I think this might be trying to make a real one? I would just reuse the `pincol` fixture here